package/metrics_daemon/metrics_daemon/src/metrics_daemon.h

```diff
@@ -0,0 +1,12 @@
+//
+// Created by Yi Yang on 9/7/18.
+//
+
+#ifndef PIKSI_BUILDROOT_METRICS_DAEMON_H
+#define PIKSI_BUILDROOT_METRICS_DAEMON_H
+
+extern "C" {
```

Comment:
This should be wrapped by `#ifdef __cplusplus` see below as an example:
https://github.com/swift-nav/piksi_buildroot/blob/e47e15952ab3c37e28497b14806f8bc2f38198f3/package/libpiksi/libpiksi/include/libpiksi/util.h#L27-L29